apps/condo/domains/organization/integrations/sbbol/sync/requestTransactions.js

```diff
@@ -124,29 +125,69 @@ async function requestTransactionsForDate ({ userId, bankAccounts, context, stat
             // WORKFLOW_FAULT means invalid request parameters, that can occur in cases:
             // when report is requested for date in future
             // when report page does not exist, for example number is out of range of available pages
-            if (get(transactions, 'error.cause') === 'WORKFLOW_FAULT') reqErrored = true
-            if (get(summary, 'error.cause') === 'WORKFLOW_FAULT') reqErrored = true
-            if (get(summary, 'error.cause') === 'DATA_NOT_FOUND_EXCEPTION') reqErrored = true
-            if (get(transactions, 'error.cause') === 'UNAUTHORIZED') {
+            if (get(transactions, 'error.cause') === SBBOL_ERRORS.WORKFLOW_FAULT) {
+                reqErrored = true
+                transactionException = SBBOL_ERRORS.WORKFLOW_FAULT
+            }
+            if (get(summary, 'error.cause') === SBBOL_ERRORS.WORKFLOW_FAULT) {
+                reqErrored = true
+                summaryException = SBBOL_ERRORS.WORKFLOW_FAULT
+            }
+
+            // DATA_NOT_FOUND_EXCEPTION means that the statement not found and cannot be generated
+            if (get(transactions, 'error.cause') === SBBOL_ERRORS.DATA_NOT_FOUND_EXCEPTION) {
+                reqErrored = true
+                transactionException = SBBOL_ERRORS.DATA_NOT_FOUND_EXCEPTION
+            }
+            if (get(summary, 'error.cause') === SBBOL_ERRORS.DATA_NOT_FOUND_EXCEPTION) {
+                reqErrored = true
+                summaryException = SBBOL_ERRORS.DATA_NOT_FOUND_EXCEPTION
+            }
```

Comment:
Is that errors happen at same time? Can you please reorganize this code for improving readability?